### PR TITLE
add anydesk

### PR DESCRIPTION
**Pull request by nakadzima9**
Added a new line at the end and refactoring code.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -40,6 +40,7 @@ __pycache__
 
 # Distribution / packaging
 .Python
+.idea/
```
Comment (asdil12) [inline .gitignore:43]: `.idea` is already part of the gitignore in the current master.

Comment (nakadzima9) [reply .gitignore:43]: Oh, sorry I didn't know.

**Comment by asdil12** [comment]
Could you squash your commits together into a single commit?

**Comment by nakadzima9** [comment]
Of course, I'll try to do it right now.

**Pull request closed without merging**